paper-tab.html

```diff
@@ -116,6 +116,18 @@
         Polymer.PaperRippleBehavior
       ],
 
+      /**
+       * Fired when the paper-tab recieves focus.
```

Comment:
We usually put the events docs at the end of the `<script>` tag.
Also, there is a typo here: `recieves` should be `receives`.